src/MassTransit.ApplicationInsights/ApplicationInsightsConsumeFilter.cs

```diff
@@ -12,100 +12,98 @@
-		private readonly Action<IOperationHolder<RequestTelemetry>, T> _configureOperation;
-		private readonly string _telemetryHeaderRootKey;
-		private readonly string _telemetryHeaderParentKey;
-
-		public ApplicationInsightsConsumeFilter(TelemetryClient telemetryClient
-			, Action<IOperationHolder<RequestTelemetry>, T> configureOperation
-			, string telemetryHeaderRootKey
-			, string telemetryHeaderParentKey
-			)
-		{
-			_telemetryClient = telemetryClient;
-			_configureOperation = configureOperation;
-			_telemetryHeaderRootKey = telemetryHeaderRootKey;
-			_telemetryHeaderParentKey = telemetryHeaderParentKey;
-		}
-
-		public void Probe(ProbeContext context)
-		{
-			context.CreateFilterScope("TelemetryConsumeFilter");
-		}
-
-		public async Task Send(T context, IPipe<T> next)
-		{
-			var messageType = context.SupportedMessageTypes.FirstOrDefault() ?? "Unknown";
-
-			// After the message is taken from the queue, create RequestTelemetry to track its processing.
-			var requestTelemetry = new RequestTelemetry
-			{
-				Name = $"{StepName} {context.ReceiveContext.InputAddress.LocalPath} {messageType}",
-			};
-
-			requestTelemetry.Context.Operation.Id = context.Headers.Get<string>(_telemetryHeaderRootKey);
-			requestTelemetry.Context.Operation.ParentId = context.Headers.Get<string>(_telemetryHeaderParentKey);
-
-			using (IOperationHolder<RequestTelemetry> operation = _telemetryClient.StartOperation(requestTelemetry))
-			{
-				operation.Telemetry.Properties.Add(MessageType, messageType);
-				operation.Telemetry.Properties.Add(QueuePath, context.ReceiveContext.InputAddress.LocalPath);
-
-				if (context.MessageId.HasValue)
-					operation.Telemetry.Properties.Add(MessageId, context.MessageId.Value.ToString());
-
-				if (context.ConversationId.HasValue)
-					operation.Telemetry.Properties.Add(ConversationId, context.ConversationId.Value.ToString());
-
-				if (context.CorrelationId.HasValue)
-					operation.Telemetry.Properties.Add(CorrelationId, context.CorrelationId.Value.ToString());
-
-				if (context.RequestId.HasValue)
-					operation.Telemetry.Properties.Add(RequestId, context.RequestId.Value.ToString());
-
-				_configureOperation?.Invoke(operation, context);
-
-				try
-				{
-					await next.Send(context).ConfigureAwait(false);
-
-					operation.Telemetry.Success = true;
-				}
-				catch (Exception ex)
-				{
-					_telemetryClient.TrackException(ex, operation.Telemetry.Properties);
-
-					operation.Telemetry.Success = false;
-					throw;
-				}
-				finally
-				{
-					_telemetryClient.StopOperation(operation);
-				}
-			}
-		}
-	}
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using GreenPipes;
+    using Microsoft.ApplicationInsights;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+
+    public class ApplicationInsightsConsumeFilter<T> : IFilter<T> where T : class, ConsumeContext

```

Comment:
This is not the proper code style. Look at the rest of the MT code base and you'll see that it should have a break after the `:` and the constraints should be on a separate line. Using R# code formatting (or Rider) should auto-format the code.